bait-files/src/main/webapp/success.jsp

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Form Successfully Submitted</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="../success.css">
+  </head>
+  <body>
+    <img src="https://img.icons8.com/cotton/128/000000/checkmark.png" alt="Green checkmark with circle around it"></img>
```

Comment:
Icons8 license requires you to link to their site for commercial use https://icons8.com/pricing

Good alternative is https://iconify.design/icon-sets/ which has icon sets with Apache and MIT licenses. It would be cool if icon was the same green in our colour scheme whatever you decide to do